test/tools/libtesteth/TestHelper.cpp

```diff
@@ -208,6 +208,18 @@ std::vector<boost::filesystem::path> getJsonFiles(std::string const& _dirPath)
 	return jsonFiles;
 }
 
+std::string executeCmd(std::string const& _command)
+{
+	char output[1024];
+	FILE *fp = popen(_command.c_str(), "r");
+	if (fp == NULL)
+		BOOST_ERROR("Failed to run " + _command);
+	if (fgets(output, sizeof(output) - 1, fp) == NULL)
+		BOOST_ERROR("Reading empty result for " + _command);
+	pclose(fp);
```

Comment:
The return value of `pclose` should be checked to see if the command returned `0`.